planning/behavior_path_planner/src/util/lane_change/util.cpp

```diff
@@ -474,15 +474,11 @@ std::pair<bool, bool> getLaneChangePaths(
       acceleration);
 
     if (is_valid && is_safe) {
```

Comment:
```suggestion
    if (is_safe) {
```

since not valid path will be skipped anyway, so we can just check for is_safe.